modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -298,14 +294,29 @@ public function enqueue_scripts() {
 			)
 		);
 
-		if ( $this->is_iframed_block_editor() ) {
-			$src_calypso_iframe_bridge = $debug
-				? '//widgets.wp.com/wpcom-block-editor/calypso-iframe-bridge-server.js?minify=false'
-				: '//widgets.wp.com/wpcom-block-editor/calypso-iframe-bridge-server.min.js';
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
This is intended for loading the `wpcom.js` script on Atomic sites, but it seems `IS_WPCOM` is not defined on those sites. Do we have any other way of performing such check?

Reply:
We have `jetpack_is_atomic_site()` that you could use to check for Atomic sites.

Reply:
Maybe [`jetpack_is_atomic_site`](https://github.com/Automattic/jetpack/blob/31efa189ad223c0eb7ad085ac0650a23facf9ef5/functions.global.php#L75)